edgelet/iotedge/src/support_bundle.rs

```diff
@@ -58,9 +58,7 @@ where
                     let path = PathBuf::from(state.output_location.get_file_location());
                     println!(
```

Comment:
I thought something about this (no direct call to display())
```rust
println!("Created support bundle at {:?}", path.canonicalize().unwrap_or_else(|_| path));
```

Reply:
![image](https://user-images.githubusercontent.com/13950901/75065962-ca128a00-549e-11ea-9ff9-0268da9e78ee.png)

Reply:
:? works, but is escapes the string ```C:\\Users\\Lee\\source``` instead of ```C:\Users\Lee\source```